File: C++/598. Range Addition II.cpp
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>>& o) {
        int i,r=m,c=n;
        for(i=0;i<o.size();i++){
            r=min(r,o[i][0]);
            c=min(c,o[i][1]);
        }
        return(r*c);
    }
};

File: JAVA/598. Range Addition II.java
class Solution {
    public int maxCount(int m, int n, int[][] o) {
        int i,r=m,c=n;
        for(i=0;i<o.length;i++){
            r=Math.min(r,o[i][0]);
            c=Math.min(c,o[i][1]);
        }
        return(r*c);
    }
}